policyengine_us/tests/policy/baseline/gov/hhs/tanf/cash/integration.yaml

```
@@ -71,6 +71,7 @@
       spm_unit:
         members: [parent, child]
         is_tanf_enrolled: true
+        dc_tanf_meets_work_requirements: true
```

```suggestion
        dc_tanf_meets_work_requirements: true
```
I added the work requirement to the dc_tanf_eligible, so this is a necessary input.